 Traceback (most recent call last):
  File "/usr/share/blender/scripts/addons/io_scene_gltf2/__init__.py", line 378, in execute

